compiler_opt/rl/imitation_learning/generate_bc_trajectories_lib.py

```diff
@@ -46,6 +46,13 @@
 from compiler_opt.distributed import buffered_scheduler
 from compiler_opt.distributed.local import local_worker_manager
 
+_BASE_PATH = flags.DEFINE_string(
```

Comment:
maybe _PERSISTENT_OBJECTS_PATH or something - "base" is very generic.

btw, could we "hitch" this path to be a subdirectory of where we save the model?

Reply:
Not completely sure how we can do this since the code that trains the model is completely separate from this. We could potentially replace this with a sub-directory of where we save the collected trajectories.